### PR TITLE
Update dependencies to support node 4.x

### LINKED ISSUE #70: Not compatible with node 4
According to @cadecairos in #69, make-valet is currently not compatible with node 4. We should get it working so we can stay on node LTS versions.

### PR DESCRIPTION
**Pull request by cadecairos**
fixes #70

### CODE REVIEW

**Comment by cadecairos** [comment]
R? @jbuck

File: server.js [hunk 1]
```diff
@@ -87,8 +87,7 @@ if ( env.get( "ENABLE_GELF_LOGS" ) ) {
   app.use(express.logger());
 }
 
-app.use(helmet.iexss());
-app.use(helmet.contentTypeOptions());
+app.use(helmet());
```
Comment (jbuck) [inline server.js]: Will this enable more security features than we want to?

Comment (cadecairos) [inline server.js]: hmm

File: package.json [hunk 2]
```diff
@@ -30,6 +30,6 @@
     "download-locales": "webmaker-download-locales make-valet"
   },
   "engines": {
-    "node": "0.10.x"
+    "node": "4.1.x"
```
Comment (jbuck) [inline package.json]: ^4.1.1"

**Comment by jbuck** [comment]
@cadecairos r+

**Pull request merged**